apps/pxweb2/src/app/app.tsx

```diff
@@ -526,12 +526,13 @@ export function App() {
     <>
       <Header />
       <div className={styles.main}>
-        <div className={styles.desktopNavigation}>
+        <div className={`${styles.desktopNavigation}`}>
           <NavigationRail
             onChange={changeSelectedNavView}
             selected={selectedNavigationView}
           />
-          {selectedNavigationView !== 'none' && (
+        {selectedNavigationView !== 'none' && (
```

Comment:
Indentation here seems off, have you formatted the files?

Reply:
Made suggested change